### PR TITLE
refactor: toggle off  LRD API (RDCC-2011)

### PR DESCRIPTION
**Pull request by abhijitdiwan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-2011

### Change description ###
refactor: toggle off  LRD API (RDCC-2011)

**Does this PR introduce a breaking change?** (check one with "x")
No

### CODE REVIEW

**Review by KOTLAPRA23 — APPROVED** [review]

**Review by shreedhar-hmcts — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/lrdapi/service/impl/FeatureToggleServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+public class FeatureToggleServiceImplTest {
+
+    LDClient ldClient = mock(LDClient.class);
+    FeatureToggleServiceImpl flaFeatureToggleService = mock(FeatureToggleServiceImpl.class);
+
+    @Test
+    public void testIsFlagEnabled() {
+        flaFeatureToggleService = new FeatureToggleServiceImpl(ldClient, "rd");
+        assertFalse(flaFeatureToggleService.isFlagEnabled("test", "test"));
+    }
```
Comment (shashank-rastogi) [inline src/test/java/uk/gov/hmcts/reform/lrdapi/service/impl/FeatureToggleServiceImplTest.java:19]: why are we using a test flag?

**Review by shashank-rastogi — CHANGES_REQUESTED** [review]

File: src/test/java/uk/gov/hmcts/reform/lrdapi/service/impl/FeatureToggleServiceImplTest.java (hunk 1 above)
Comment (abhijitdiwan) [reply src/test/java/uk/gov/hmcts/reform/lrdapi/service/impl/FeatureToggleServiceImplTest.java:19]: tese are unit test.

Comment (abhijitdiwan) [reply src/test/java/uk/gov/hmcts/reform/lrdapi/service/impl/FeatureToggleServiceImplTest.java:19]: done

Comment (abhijitdiwan) [reply src/test/java/uk/gov/hmcts/reform/lrdapi/service/impl/FeatureToggleServiceImplTest.java:19]: done

**Review by shashank-rastogi — APPROVED** [review]

**Pull request merged**